addon-test-support/load.js

```diff
@@ -19,4 +22,73 @@ export default function loadEmberExam() {
   if (window.Testem) {
     patchTestemOutput(TestLoader);
   }
+
+  testLoader = new TestLoader();
 }
+
+// loadTests() is equivalent to ember-qunit's loadTest() except this does not create a new TestLoader instance
+function loadTests() {
+  if (testLoader === undefined) {
+    new Error('testLoader is not defined. It must call `loadEmberExam()` before calling `loadTest()`.')
+  }
+
+  testLoader.loadModules();
+}
+
+// setupQUnitCallBacks() registers QUnit callbacks neeeded for the load-balance option.
+function setupQUnitCallbacks(qunit) {
+  if (!location.search.includes('loadBalance')) {
+    return;
+  }
+  qunit.begin(function() {
+    return new self.Promise(function(resolve) {
+      Testem.on('testem:next-module-response', function getTestModule(moduleName) {
```

Comment:
the `begin` and `moduleDone` callbacks appear extremely similar, should they be unified?